core/templates/dev/head/domain/collection/CollectionNodeObjectFactory.js

```diff
@@ -17,20 +17,30 @@
  * collection node domain objects.
  */
 
-// TODO(bhenning): Implement validation functions and related tests.
+// TODO(bhenning): This should be provided by the backend.
+oppia.constant('ACTIVITY_STATUS_PRIVATE', 'private');
+
 oppia.factory('CollectionNodeObjectFactory', [
-      'SkillListObjectFactory', function(SkillListObjectFactory) {
+      'SkillListObjectFactory', 'ACTIVITY_STATUS_PRIVATE',
+      function(SkillListObjectFactory, ACTIVITY_STATUS_PRIVATE) {
+    var _initializeExplorationDetails = function(
+        node, explorationSummaryBackendObject) {
+      node._explorationTitle = explorationSummaryBackendObject.title;
```

Comment:
Seems a bit weird to set this even if the summary doesn't exist. Why do we have it at all (and _explorationIsPrivate)? They're all in the summary object anyway...